test/integration/run-integration-tests.sh

```diff
@@ -0,0 +1,3 @@
+for element in *-suite
+   do watai "$element"
+done
```

Comment:
You need to exit with a nonzero exit code if any of the execution returns a nonzero exit code.